common/src/main/java/io/netty/util/HashedWheelTimer.java

```diff
@@ -368,7 +369,8 @@ public void start() {
         }
 
         if (leak != null) {
-            leak.close();
+            boolean closed = ResourceLeakDetector.close(leak, this);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "closed". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1854)